server/.env.testing

```diff
@@ -0,0 +1,4 @@
+ENV=testing
```

Comment:
TODO: automate with deploy script when https://passculture.atlassian.net/browse/PC-14035 is DONE